Userland/Services/LoginServer/main.cpp

```diff
@@ -99,13 +99,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         login(account.value(), *window);
     };
 
-    char const* auto_login = nullptr;
+    StringView auto_login;
 
     Core::ArgsParser args_parser;
     args_parser.add_option(auto_login, "automatically log in with no prompt", "auto-login", 'a', "username");
     args_parser.parse(arguments);
 
-    if (!auto_login) {
+    if (auto_login.is_null()) {
```

Comment:
Use `is_empty()` instead.

Reply:
Sorry, I should have pay attention to that, you already mentioned it to me!